src/pages/Products.jsx

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import ProductFilter from "../components/products/ProductFilter";
+import TopBar from "../components/products/TopBar";
+import IconWrapper from "../components/shared/IconWrapper";
+import { useProducts } from "../context/products/ProductContext";
+import { FiHeart } from "react-icons/fi";
+const Products = () => {
+  const { products } = useProducts();
+  console.log(products[0]);
+  return (
+    <main className="mx-2 flex container">
+      <ProductFilter />
+      <section className="mx-2 border product-listing">
+        <TopBar />
+        <div className="flex wrap center products">
+          {products.map((product) => (
+            <div
```

Comment:
you may use `<ul>` and `<li>` tags for the list. Any reason to use only divs?

Reply:
These are product cards, so `<ul> ` `<li> didn't come to my mind.